editor/script_editor_debugger.cpp

```diff
@@ -612,7 +612,9 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 
 			if (!EditorNode::get_log()->is_visible()) {
 				if (EditorNode::get_singleton()->are_bottom_panels_hidden()) {
-					EditorNode::get_singleton()->make_bottom_panel_item_visible(EditorNode::get_log());
+					if (EDITOR_DEF("run/output/always_open_output_on_play", true)) {
```

Comment:
Shouldn't it be `EDITOR_GET`? It's already defined (actually twice) at this point.

Reply:
You're right. Changed.